src/packages/@ncigdc/modern_components/SCRNASeq/SCRNASeq.js

```diff
@@ -22,9 +23,14 @@ const dataTypes = Object.keys(stubData);
 const enhance = compose(
   setDisplayName('EnhancedSCRNASeq'),
   pure,
+  withHandlers({
+    handleAnalysisClick: () => {
+      console.log('clicked analysis dropdown item');
```

Comment:
pending stuff, I imagine?

Reply:
yeah we don't know how to handle the clicking yet, but I didn't want to use the toolbar click function and have it crash.